Text draft for narrative episode (#17)
carpentries/lesson-development-training: episodes/13-narrative.md

exercises: 50
---

::::::::::::::::::::::::::::::::::::::: objectives

- Find candidate datasets to use in a lesson.
- Evaluate the suitability of a dataset to be used in a lesson.
- Choose examples that will prepare learners for formative assessments in the lesson.
- Develop a story

::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::: questions

- Why should a lesson tell a story?
- What considerations are there when choosing an example dataset for a lesson?
- Where can I find openly-licensed, published data to use in a lesson?

::::::::::::::::::::::::::::::::::::::::::::::::::

## Creating a narrative

<!--- Should this section go before LO and questions? -->

Writing your lesson as a story helps learners stay motivated and engaged.

> The story you create can also help learners more easily connect how the skills they
> are learning to how the skills could be useful after the workshop.

Reviewer: ```suggestion
The story you create can also help learners more easily connect how the skills they
are learning now could be useful after the workshop.
```

You can enable learners to make connections between the what they learn in your lesson to their
own work by creating a narrative that resembles a situation the learners might
encounter in their own work.
Depending on the tool you are teaching, you might also include a particular dataset as a part of the story you are weaving into your lesson.
It is common for lessons to include a dataset that is used in examples and discussed throughout.
This can help you maintain a narrative flow and make the lesson feel more authentic
Even if your topic doesn't require a dataset, deciding on a consistent narrative will
help create a flow between lessons and reduce cognitive load for learners.

## Finding a dataset

When searching for a dataset to use in your lesson, 
there are a number of factors to consider.
First, if possible you want the dataset to be an authentic use
case for researchers in your target audience.
This means balancing the size and complexity of a dataset
while avoiding additional cognative load for learners.
The dataset may need a certain number of observations
to demonstrate some of the skills you are teaching or have a

> realistic number of variables to what learners many encounter

Reviewer: ```suggestion
number of variables that is sufficiently similar to what learners many encounter
```

in their work to feel authentic.
At the same time, you want your learners to be able to interpret the
data fairly easily during the lesson so they can focus on the skills
you are teaching.
This may mean you will need to subset your dataset or remove variables
observed.

> You may want to also include and review a data dictionary in your lesson,
> explicitly taking the time to review the information included in the
> dataset.

Author: ```suggestion
You may want to also include and review a data dictionary in your lesson,
explicitly taking the time to review the information included in the
dataset.
For inspiration, see the [Social Sciences Data Carpentry data dictionary](https://datacarpentry.org/socialsci-workshop/data/).
```

> dataset.

Author: Changed to include link to data DC SS data dictonary

each dataset considered.  *Does the data contain personally identifiable information?*
*Was the data collected without permission from the groups or individuals included?*
*Will the data be upsetting to learners in the workshop?*
If the answer to any of these questions might be yes, you will need to do more
research on it and continue to look for other options.  Commonly data is misused
from historically excluded and exploited groups.
The [CARE Principles for Indigenous Data Governance](http://doi.org/10.5334/dsj-2020-043) (Collective Benefit, Authority to Control, Responsibility, and Ethics) are good starting point
for thinking about data sovereignty and considering the ethics of data collected
about individual or groups of people.

### Examples of Public Repositories

When looking for data to reuse, consider public repositories in the subject area
for your lesson or general data repositories such as:
  - [Dryad](https://datadryad.org/)
  - [The Data Cuartion Network's datasets](https://datacurationnetwork.org/datasets/)
  - [The Offical Portal for European Data](https://data.europa.eu/)
  - [DataONE](https://www.dataone.org/)

> 

Reviewer 2: ```suggestion
- [The Official Portal for Argentina Data](https://www.datos.gob.ar/) - (In Spanish)
```

Reviewer 2: Is it worth mentioning that there are packages with ready-to-teach datasets? And that some have translations?

Like Palmer Penguins dataset and the datos/dados packages (which are Spanish and Portuguese translations of the datasets used in the R4DS book).

Reviewer 2: We will only list official portals? Because Tidy Tuesday and #DatosDeMiércoles (the Latin American cousin of Tidy Tuesday) have nice datasets that can be used for teaching.

>   - [The Offical Portal for European Data](https://data.europa.eu/)
>   - [DataONE](https://www.dataone.org/)
> - [The Official Portal for Argentina Data](https://www.datos.gob.ar/) - (In Spanish)

Author: ```suggestion
  - [The Offical Portal for European Data](https://data.europa.eu/)
  - [The Official Portal for Argentina Data](https://www.datos.gob.ar/) - (In Spanish)
  - [DataONE](https://www.dataone.org/)
  
You may also want to look for ready to look for packages or built-in data from the tool you are teaching.
For example, [Palmer Penguins](https://cran.r-project.org/web/packages/palmerpenguins) is commonly used when teaching R.
[Tidy Tuesday](https://github.com/rfordatascience/tidytuesday) or [Datos de Miércoles](https://github.com/cienciadedatos/datos-de-miercoles) are 
also good places to look for new datasets that meet your criteria.
```

Author: @yabellini Does this addition address your suggestion?  Also is the link and capitalization for Datos de Miércoles correct?

Reviewer 2: Yes and yes, thank you!

Even with a CC0 license, you will still want to follow best practice in 
giving attribution to the data provider or collecting agency.

> :::::::::::::::: callout

Reviewer 3: Nice! Does this mean that the div validator helped you out?

Author: Since I was getting that [other fenced div issue](https://github.com/carpentries/pegboard/issues/83), I actually found it myself looking for what might have caused that issue.
